packages/components/pie-switch/src/switch.scss

```diff
@@ -48,6 +48,10 @@
 
     &:hover {
         background-color: hsl(var(--dt-color-interactive-form-h), var(--dt-color-interactive-form-s), calc(var(--dt-color-interactive-form-l) - var(--dt-color-hover-01)));
+
+        @supports (background-color: color-mix(in srgb, black, white)) {
```

Comment:
I have mixed feelings about this. I like that the mixin you've written handles both hover and active states, but I wonder if having separate mixins for hover and active means that these form components could use the hover one without needing the active one?

The downside would be a bit more complexity in the other components which currently use the mixin for both hover and active states. What do you think?

Reply:
Actually for switch I should be able to use the mixin, I'll update.

But actually I had the exact thought. Why I left it as is because the amount of components that use both hover and active styles is bigger than the ones that have only hover so I thought about those as more of exceptions. Plus hover and active styles go in sync and if we will have hover and active separate there will be a risk of passing different incorrect hover/active token, like for hover 01 and for active 02. And also for form elements the styles sometimes apply to the child of hovered element. For example `&:hover { .c-checkbox-tick {...}}` that doesn't align with the shared mixin so it was another point of keeping it separate.